storybook/index.html

```diff
@@ -329,9 +329,18 @@ <h1 class="sectionHeading">BAMP</h1>
                 <section class="demo" data-module-name="navModal">
                     <p class="navModal--p">Only available in Mobile</p>
                     <div class="navModal">
-                        <h1 class="navModal--h1">x</h1>
-                        <h2 class="navModal--h2">Home <br> Artwork <br> About Us <br> Programs <br> Events <br> Services <br> Shop <br> Get Involved</h2>
-                        <div class="navModal--btn">
+                        <h1 class="navModal--h1">×</h1>
```

Comment:
we should use a `button` element for this (and the class could be `navModal--close` or `navModal--closeBtn` etc)